src/components/functional/ledger-connect/components/ledger-addresses/ledger-addresses.scss

```diff
@@ -44,39 +45,41 @@
         }
 
         .ledger-addresses-list-item {
-          @apply mvx:rounded-lg mvx:flex mvx:items-center mvx:text-white mvx:p-4 mvx:h-16 mvx:w-full;
+          @apply mvx:rounded-lg mvx:flex mvx:items-center mvx:p-4 mvx:h-16 mvx:w-full;
           @apply mvx:relative mvx:cursor-pointer mvx:transition mvx:duration-200 mvx:ease-in-out;
 
           border: 1px solid transparent;
+          color: var(--mvx-text-color-primary);
 
           &:before {
             @apply mvx:absolute mvx:left-0 mvx:top-0 mvx:bottom-0 mvx:right-0 mvx:pointer-events-none mvx:rounded-lg mvx:opacity-30;
-            background-color: #353535;
+            background-color: var(--mvx-bg-color-secondary);
             content: '';
           }
 
           &.checked,
           &:hover {
-            border-color: #23f7dd;
+            border-color: var(--mvx-text-accent-color);
           }
 
           .ledger-addresses-list-item-checkbox {
             @apply mvx:relative mvx:z-1 mvx:cursor-pointer mvx:transition mvx:duration-200 mvx:rounded-full;
             @apply mvx:ease-in-out mvx:mr-2 mvx:min-w-4 mvx:max-w-4 mvx:h-4;
-            border: 1px solid #ffffff;
+            border: 1px solid var(--mvx-text-color-primary);
 
             &:after {
               @apply mvx:absolute mvx:left-1/2 mvx:top-1/2 mvx:rounded-full mvx:opacity-0 mvx:transition-all mvx:duration-200;
               @apply mvx:ease-in-out;
               transform: translate(-50%, -50%);
-              background-color: #23f7dd;
+              transition-timing-function: ease;
```

Comment:
This needs to be removed, we already have the `mvx:ease-in-out` class above which is overwritten by this property.

Reply:
Awesome. 👍